05/app.js

```diff
@@ -1,53 +1,87 @@
 const stats = {
-    paragraphs: {
-        'p1': 0,
-    },
-    links: {
-        '/dolor.html': 0,
-    }
+  paragraphs: {
+    p1: 0,
+  },
+  links: {
+    "/dolor.html": 0,
+  },
 };
 
 /* tutaj umieść swój kod */
+const links = document.querySelectorAll(".link");
+const preventFollowLink = function (e) {
+  e.preventDefault();
+};
+links.forEach(function (link) {
+  link.addEventListener("click", preventFollowLink);
+});
+let dolorCounter = 0;
+const pTags = document.querySelectorAll("p");
 
+const countClicks = function (e) {
+  e.stopImmediatePropagation();
 
-/* nie modyfikuj kodu poniżej, ale przeanalizuj go */
+  if (
+    e.target.tagName === "A" &&
+    e.target.getAttribute("href") === "/dolor.html"
+  ) {
+    stats.links["/dolor.html"] = ++dolorCounter;
```

Comment:
Zbyt konkretnie zapisane. Co jeśli ktoś kliknie w inny link lub za dzień czy dwa adresy się zmienią? Należy dynamicznie to zrobić jak niżej.